### DISCUSSION TITLE
klog icon

### DISCUSSION THREAD

**Discussion by jotaen** [jotaen/klog#392] [Something else]
Would be cool to have an custom icon for `.klg` files, e.g. to display in file browsers.

Thoughts and suggestions are welcome.

**Comment by jotaen** [comment]
Tada 🎉 
Variations here for download: https://klog.jotaen.net/logo

![klog logo](https://klog.jotaen.net/logo/klog-black.png)

cc @vladdeSV (https://github.com/vladdeSV/vscode-klog/issues/2)

**Comment by jotaen** [comment]
And, before I forget: kudos to [Charisse](https://dribbble.com/charisseysabel) for the design idea! (@charisseysabel) Thank you! 🥇 💯

**Comment by vladdeSV** [comment]
Looks amazing! 🔥

**Comment by charisseysabel** [comment]
You're welcome 🤝
